src/app/components/organism/hero-social-bar/hero-social-bar.component.css

```
@@ -1,63 +1,28 @@
-    0deg,
-    rgb(26 0 70 / 10%) 0%,
-    rgb(11 0 30 / 80%) 100%
-  );
-  position: relative;
-  box-shadow: 19px 8px 10px 5px rgba(0, 0, 0, 0.2), inset 0 -12px 40px #3800bd26;
+  gap: 1rem;
 }
 
-::ng-deep a.social-link {
-  scale: 0.865;
+::ng-deep a.social-link .social-icon {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)